### ISSUE TITLE
Convienient Effects do not seem to work with systems other than DnD

### ISSUE THREAD

**Issue by picollo-24** [DFreds/dfreds-convenient-effects#326]
### Describe the Bug

I tried enabling this module in multiple systems other than DnD, and I never ever get the option to do it- system doesn't appear on the list of modules to be enabled.

Test has been done on latest version 11 release of Foundry, using Forge hosting.

Systems tested:

SWADE  3.3.11 (and previous sub versions- first attempt was at 3.3)

Powered by the Apocalypse 0.9.7

Warhammer Age of Sigmar: Soulbound 6.1.0

Project FU 2.3.6 (https://github.com/League-of-Fabulous-Developers/FoundryVTT-Fabula-Ultima/releases/tag/2.3.6)

I know that module is technically compatibile (and easily can be enabled) for last module, as someone described workaround how to enable it (link can be found here: https://discord.com/channels/1194506991754805278/1231056580061958174/1231056580061958174 )
Long story short-  Adding relationship in  Foundry manager allows to activate this module in Project FU.

However this is not native support mentioned in the description of module, so I consider it a bug.

**Comment by DFreds** [comment]
Yep, it's a hard dependency. The whole module is being rewritten to remove that though.
